admin/config.yml

```diff
@@ -49,8 +52,59 @@ collections:
         file: "_data/team-alpha.json"
         description: "Team Alpha Members List"
         fields:
-          - {label: "Name", name: "name", widget: "string"}
-          - {label: "Rule", name: "rule", widget: "string"}
-          - {label: "Song", name: "song", widget: "string"}
-          - {label: "Person", name: "person", widget: "string"}
-          - {label: "Fact", name: "fact", widget: "string"}
+          - label: Entries
+            name: Entries
+            widget: list
+            fields:
+              - {label: Name, name: name, widget: string}
+              - {label: Rule, name: rule, widget: string}
+              - {label: Song, name: song, widget: string}
+              - {label: Person, name: person, widget: string}
+              - {label: Fact, name: fact, widget: string}
+              - {label: Team, name: team, widget: string}
+      - name: "alumni-bootcamp"
+        label: "Alumni Bootcamp"
+        file: "_data/alumni-bootcamp.yml"
+        description: "Bootcamp Alumni List"
+        fields:
+          - label: Entries
+            name: Entries
+            widget: list
+            fields:
+              - {label: Name, name: name, widget: string}
+              - {label: Website, name: website, widget: string}
+              - {label: Github, name: github, widget: string}
+              - {label: Linkedin, name: linkedin, widget: string}
+              - {label: Image, name: image, widget: string}
+              - {label: Position, name: position, widget: string}
+              - {label: Cohort, name: cohort, widget: string}
+      - name: "events"
```

Comment:
@mhisle22 I'd like to keep events as its own collection instead of in the data folder. We can talk about this later if you want.